dipy/tracking/streamline.py

```diff
@@ -113,3 +116,169 @@ def select_random_set_of_streamlines(streamlines, select):
     len_s = len(streamlines)
     index = np.random.randint(0, len_s, min(select, len_s))
     return [streamlines[i] for i in index]
+
+
+def reduce_rois(rois, include):
```

Comment:
It is not clear to me which functions should be in the ``dipy.tracking.streamline`` and which in ``dipy.tracking.utils``. How did you decide which should go where? Did you have something specific in mind?